src/zabapgit_object_clas.prog.abap

```diff
@@ -621,6 +623,7 @@ CLASS lcl_object_clas IMPLEMENTATION.
           lt_tpool      TYPE textpool_table,
           lv_object     TYPE dokhl-object,
           lv_state      TYPE dokhl-dokstate,
+          lt_seocompotx TYPE ty_seocompotx_tt,
```

Comment:
Should this be renamed to lt_class_components or is it too generic? Often in ABAP I prefer to name things like instead of using "lv_bukrs", I use "lv_company_code".

But, in this case, does it makes more clear or not?